sample/src/main/java/io/ona/kujaku/sample/activities/CardActivity.java

```diff
@@ -45,6 +45,9 @@ public void onClick(View v) {
                 isCardVisible = !isCardVisible;
             }
         });
+
+        ImageButton btnHamburgerMenu = findViewById(R.id.map_view_hamburger_menu);
+        boolean isTablet = getResources().getBoolean(io.ona.kujaku.R.bool.isTablet);
```

Comment:
Remove this, I also don't think it is needed any longer

Reply:
Done.